translator/translate/util/placeholderUtil.go

```diff
@@ -35,6 +37,7 @@ func ResolvePlaceholder(placeholder string, metadata map[string]string) string {
 	for k, v := range metadata {
 		tmpString = strings.Replace(tmpString, k, v, -1)
 	}
+	tmpString = strings.Replace(tmpString, datePlaceholder, time.Now().Format("2006-01-02"), -1)
```

Comment:
It's not flexible to use placeholder {date} for a specific format, i.e. ISO-8601, because someone else may prefer another format for {date}. It will be more generic to support additional "customized suffix" for logstream.

Reply:
I can understand the the desire to make the date format customizable, but would it make sense for ISO-8601 to at least be the default? Zero-padded, descending values is the ideal format when dealing with sorting by strings, such as browsing in the AWS console.

Anyway, for the formatting itself are you requesting that this be moved to a new property in the JSON config file, and the property's value being the desired format?
```json
{
    "file_path": "/var/log/apache2/access.log",
    "log_group_name": "/aws/ec2/{local_hostname}/logs",
    "log_stream_name": "access.log",
    "log_stream_name_suffix": "2006-01-02"
},
```
In this way, the separator may be an issue.  I'm not extremely familiar with the date formatting string used in Golang, but it seems like the format is somewhat limited; for example I don't see how we could use a numeral as a separator.

Another alternative that may be worth considering would be extending the template match string to include the date format string, like this:
```json
{
    "file_path": "/var/log/apache2/access.log",
    "log_group_name": "/aws/ec2/{local_hostname}/logs",
    "log_stream_name": "access.log-{date:2006-01-02}",
},
```
This is where the default mentioned above would come into play; the template match could select an optional date format string (selecting any characters after a `:` and continue until the next `}'), and use that in the date generation string.  Of the two options, I think this one is more flexible, as the user could decide if the date string should be a prefix or suffix.